src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -3,13 +3,14 @@
 PeakDetectorCLI::PeakDetectorCLI() {
 	parseOptions = new ParseOptions();
 	_pollyIntegration = new PollyIntegration();
+    alignType = NO_TYPE;
 }
 
 void PeakDetectorCLI::processOptions(int argc, char* argv[]) {
 
 	//command line options
 	const char * optv[] = {
-							"a?alignSamples: Enter non-zero integer to run alignment <int>",
+                            "a?alignSamples: 1=obi-warp, 2=poly fit",
```

Comment:
"Enter 1 for obi-warp alignment, 2 for polyfit"
Consistent with the other help messages.

Reply:
yes, makes sense